modules.yml

```diff
@@ -1050,3 +1050,14 @@
   path: import/nmap
   required_keys: []
   version: '1.0'
+- author: Ryan Hays
+  dependencies: []
+  description: Compares the hosts table and removes hosts that are not found within
+    the provided scope document. Scope should be defined within the document either
+    by single IP or in CIDR notation (eg. 192.168.1.0/24)
```

Comment:
Why not by domain? In web app land, scope is usually by domain rather than IP. Just a thought.

Reply:
Could do either..... I supposed network land I usually get an IP scope :). Could do two modules one for IP and one for the domain.

Reply:
You don't like the idea of dynamically detecting a scope entry as IP, CIDR, or domain, then applying as needed? You're already doing it for IP and CIDR, so adding another module seems inconsistent, no?

Reply:
Just felt like a lot of the other modules are broken up based on that at least that is how I have noticed it. Shouldn't be too hard to add in looking at the domains as well.

Reply:
Updated this for reading in a domain now within the scope document.